app/views/coronavirus_landing_page/one_line_break.html.erb

```diff
@@ -0,0 +1,79 @@
+<% add_view_stylesheet("covid") %>
+
+<% content_for :meta_tags do %>
+  <%= @requested_variant.analytics_meta_tag.html_safe %>
+<% end %>
+
+<%= render partial: "coronavirus_landing_page/components/shared/meta_tags", locals: {
+  metadata: details.metadata,
+  corona_opengraph_images: true
+} %>
+
+<%= render partial: 'coronavirus_landing_page/components/shared/schema', locals: {
+  presenter: details,
+  special_announcement: special_announcement,
+} %>
+
+<%= render partial: 'coronavirus_landing_page/components/landing_page/page_header', locals: {
+  details: details,
+  breadcrumbs: breadcrumbs,
+  title: details.title,
+  header: details.header_section
+} %>
+
+<div class="govuk-width-container covid__page" data-module="coronavirus-landing-page">
+  <section class="covid__section covid__section--first">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-one-third">
+        <%= render "govuk_publishing_components/components/heading", {
+          text: "Guidance and support",
+          margin_bottom: 3,
+          font_size: "m"
+        } %>
+        <%= render partial: 'coronavirus_landing_page/components/shared/country_guidance', locals: { guidance: details.additional_country_guidance } %>
+      </div>
+
+      <div class="govuk-grid-column-two-thirds">
+        <%= render partial: 'coronavirus_landing_page/components/shared/accordion_sections', locals: {
+          accordions: details.sections
+        } %>
+        <%= render partial: 'coronavirus_landing_page/components/shared/country_section' %>
+      </div>
+    </div>
+  </section>
+
+  <section class="covid__section">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-one-third">
+        <%= render "govuk_publishing_components/components/heading", {
+          text: "More COVID-19 information",
+          margin_bottom: 6,
+          font_size: "m"
+        } %>
+      </div>
+
+      <div class="govuk-grid-column-two-thirds">
+        <%= render partial: 'coronavirus_landing_page/components/shared/topic_section', locals: {
+          topic_section: details.topic_section,
+          tracking_category: "pageElementInteraction",
+          tracking_action: "CoronavirusInformation",
+        } %>
+      </div>
+    </div>
+  </section>
+
+  <%= content_tag :section, class: "covid__topic-wrapper" do
+    render 'govuk_publishing_components/components/signup_link', {
+      link_text:  "Sign up to get emails when we change any coronavirus information on the GOV.UK website",
+      link_href: "/email-signup?topic=/coronavirus-taxon",
+      heading: "Stay up to date with GOV.UK",
+      data: {
+        "module": "gem-track-click",
+        "track-category": "emailAlertLinkClicked",
+        "track-action": "/coronavirus",
+      },
+      background: true
+    }
+  end %>
+  <br>
```

Comment:
yep, as I mentioned above - if this is the only change, it might be easier to put this all in the show view with something like:
```
<% if @requested_variant.variant?("A") %>
<br>
<% end %>
```
and likewise for the `B`. It might make this PR a fair bit smaller as you can ditch the two new views and the switching logic from the controller. 

The meta tag section is the same across all three, so that should be fine.

(I haven't exhaustively checked that the views otherwise look the same, but I think they are)